dll/win32/newdev/newdev.c

```diff
@@ -389,9 +402,8 @@ SearchDriverRecursive(
             {
                 wcscpy(LastDirPath, DirPath);
 
-                if (wcslen(DirPath) > MAX_PATH)
-                    /* Path is too long to be searched */
-                    continue;
+                /* if wcslen(DirPath) >= MAX_PATH here it's too late, program has already crashed */
+                assert(wcslen(DirPath) < MAX_PATH);
```

Comment:
Other than this, I have no objections but I'm curious how the original `continue` path would cause the crash. Note that `assert` will not trigger in release builds.

Reply:
It wasn't the "continue" that crashed, but the "wcscat" above, because we have DirPath[MAX_PATH]